app/javascript/packs/articlePage.jsx

```diff
@@ -42,9 +34,16 @@ const userDataIntervalID = setInterval(async () => {
       } = await import('../CommentSubscription');
 
       const { articleId } = document.getElementById('article-body').dataset;
-      const { config: subscriptionType } = await getCommentSubscriptionStatus(
-        articleId,
-      );
+      
+      let subscriptionType = 'not_subscribed';
+
+      if (userStatus === 'logged-in' && user !== null) {
+        const commentSubscriptionStatus = await getCommentSubscriptionStatus(
+          articleId,
+        );
+        subscriptionType = commentSubscriptionStatus.config;
+      }
```

Comment:
Could we use destructuring here?

```suggestion
      if (userStatus === 'logged-in' && user !== null) {
        ({ config: subscriptionType } = await getCommentSubscriptionStatus(
          articleId,
        ));
      }
```

I have not tested this suggestion, so if you didn't destructure this for a reason, disregard my suggestion :smile: